### ISSUE TITLE
Choice of loss function used

### ISSUE THREAD

**Issue by Mayur28** [joonson/syncnet_trainer#17]
Hi,

I am looking through this repo and I am confused about the choice of loss function used. I am using SyncNet to measure lip-sync error and considering that this may be considered a binary classification problem, I am confused as to why the CrossEntropyLoss is used as opposed to the Binary Cross Entropy Loss.

Any clarification would be highly appreciated.
Thanks
